website/thabloid/static/thabloid/js/main.js

```diff
@@ -19,11 +19,12 @@ function openThabloidFancybox(e, btn) {
     });
 }
 
-window.onload = (event) => {
+window.addEventListener('load', (event) => {
```

Comment:
```suggestion
window.addEventListener('DOMContentLoaded', (event) => {
```

Reply:
do we actually need this? the script should be at the bottom of the html body

Reply:
jelle notified me that the window.onload can only be used once, so it would be bad practice having it in our codebase

Reply:
no I meant, do we even need the onload at all.